MATLAB/Algorithms/FDK.m

```diff
@@ -116,4 +121,23 @@
     end
 end
 
+end
+
+function proj = apply_wang_weights(proj, geo)
+
+    overlap_in_mm = geo.sDetector(1)/2 - abs(geo.offDetector(1));
+    overlap_in_pix = round(overlap_in_mm / geo.dDetector(1));
+
+    t_in_mm = linspace(-overlap_in_mm, overlap_in_mm, 2 * overlap_in_pix);
+    R_in_mm = geo.DSO;
+    w = 0.5 * (sin((pi * atan(t_in_mm / R_in_mm)) / (2 * atan(overlap_in_mm / R_in_mm))) + 1);
+    w = repmat(w, [geo.nDetector(1), 1, size(proj, 3)]);
+
+    wang_weights = ones(size(proj));
+    wang_weights(:, 1:size(w, 2), :) = w;
+    if sign(geo.offDetector(1)) < 0

```

Comment:
Similar to the comment above (and not only here), we need to think about what happens if someone adds a varying offDetector (per projection). Do we compute `overlap_in_mm` for each offset, or do we just use the first value? 

I genuinely don't know, so I am happy to get input in here. If we just take the first value, then maybe its worth adding a warning in the cases that there are more informing the user that only the first will be used.